reference/5.1/Microsoft.PowerShell.Core/Wait-Job.md

```diff
@@ -54,16 +53,23 @@ Wait-Job [-Any] [-Timeout <Int32>] [-Force] [-Filter] <Hashtable> [<CommonParame
 
 ## DESCRIPTION
 
-The `Wait-Job` cmdlet waits for Windows PowerShell background jobs to finish before it displays the
-command prompt. You can wait until any background job is complete, or until all background jobs are
-complete, and you can set a maximum wait time for the job.
+The `Wait-Job` cmdlet waits for a job to be in a terminating state before it displays the command
+prompt. The terminating states are:
```

Comment:
```suggestion
The `Wait-Job` cmdlet waits for a job to be in a terminating state before continuing execution.
The terminating states are:
```

Reply:
My vscode crashed right after our conversation and lost this change for some reason. this was originally corrected. Thanks for the check.